lib/ChainwebDb/Types/Transfer.hs

```diff
@@ -0,0 +1,66 @@
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE ImpredicativeTypes #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+{-# OPTIONS_GHC -fno-warn-missing-signatures #-}
+
+module ChainwebDb.Types.Transfer where
+
+----------------------------------------------------------------------------
+import BasePrelude
+import Data.Text (Text)
+import Data.Time.Clock (UTCTime)
+import Database.Beam
+------------------------------------------------------------------------------
+import ChainwebDb.Types.Block
+import ChainwebDb.Types.Common
+------------------------------------------------------------------------------
+data TransferT f = Transfer
+  {
+    _tr_creationtime :: C f UTCTime
+  , _tr_block :: PrimaryKey BlockT f
+  , _tr_requestkey :: C f ReqKeyOrCoinbase
+  , _tr_chainid :: C f Int64
+  , _tr_height :: C f Int64
+  , _tr_idx :: C f Int64
+  , _tr_qualName :: C f Text
```

Comment:
We should be able to ditch qualName from this table since we're filtering to TRANSFER and we already have modulename.

Reply:
Yeah that makes sense.